search/Longest_Increasing_Subsequence_using_binary_search.cpp

```diff
@@ -0,0 +1,52 @@
+#include <cassert>   /// for std::assert
```

Comment:
needs a header comment explaining the algorithm, if relevant math formulas in latex format, authors name

this is the format;
```cpp
/**
 * @file
 * @brief Add one-line description here. Should contain a Wikipedia
 * link or another source explaining the algorithm/implementation.
 * @details
 * This is a multi-line
 * description containing links, references,
 * math equations, etc.
 * @author [Name](https://github.com/handle)
 * @see related_file.cpp, another_file.cpp
 */
 ```

Reply:
done